src/lib/Scenes/City/Components/SavedEventSection/index.tsx

```diff
@@ -17,6 +17,10 @@ export class SavedEventSection extends Component<any> {
     SwitchBoard.presentNavigationViewController(this, `/city-save/${this.props.citySlug}`)
   }
 
+  navigateToBMWArtGuide() {
+    SwitchBoard.presentNavigationViewController(this, "https://www.bmw-arts-design.com/bmw_art_guide")
```

Comment:
This url is available from MP as:

```graphql
{
  city(slug: "your-slug-here") {
    name
    sponsoredContent {
      artGuideUrl
    }
  }
}
```

Reply:
We do already include this in the `GlobalMap` query, so it should be accessible there. The CityPicker is trickier because of how we present our components. @ashleyjelks can you ping me and we'll pair on it?